updater/internal/install/install.go

```diff
@@ -58,9 +61,27 @@ func (i Installer) Install(rb rollback.ActionAppender) error {
 	rb.AppendAction(action.NewServiceStopAction(i.svc))
 	i.logger.Debug("Service stopped")
 
+	// If JMX jar exists outside of install directory, make sure that gets backed up
+	jarPath := path.SpecialJMXJarFile(i.installDir)
+	jarDirPath := path.SpecialJarDir(i.installDir)
+	latestJarPath := path.LatestJMXJarFile(i.latestDir)
+	_, err := os.Stat(jarPath)
+	switch {
+	case err == nil:
+		if err := installFile(i.logger, latestJarPath, jarDirPath, i.backupDir, rb); err != nil {
+			return fmt.Errorf("failed to install JMX jar from latest directory: %w", err)
+		}
+		// Just log this error as the worst case is that there will be two jars copied over
+		if err = os.Remove(latestJarPath); err != nil {
+			i.logger.Warn("Failed to remove JMX jar from latest directory", zap.Error(err))
+		}
+	case !errors.Is(err, os.ErrNotExist):
+		return fmt.Errorf("failed determine where currently installed JMX jar is: %w", err)
+	}
```

Comment:
Any chance we can break this logic out into it's own function since it's specific to the JMX jar?